includes/definitions/discovery/vrp.yaml

```diff
@@ -14,7 +14,6 @@ modules:
             -
                 oid: hwEntityStateEntry
                 value: hwEntityCpuUsage
-                num_oid: '.1.3.6.1.4.1.2011.5.25.31.1.1.1.1.5.{{ $index }}'
                 descr: '{{ $entPhysicalName }} Processor'
```

Comment:
This will ensure num_oid computing does not get broken (short term) until more YAML files uses it.